etn-ccis/blui-react-workflows
# Feature/4131 verifycode screen

Fixes # Verify code scree page updated and completed unit test cases.



#### Changes proposed in this Pull Request:

-
-
-



#### Screenshots / Screen Recording (if applicable)

- 
<img width="527" alt="Screenshot 2023-06-28 at 12 48 42 PM" src="https://github.com/etn-ccis/blui-react-workflows/assets/133086327/db659c25-b5df-4205-b000-aa7e441f1cee">

<img width="527" alt="Screenshot 2023-06-28 at 12 48 57 PM" src="https://github.com/etn-ccis/blui-react-workflows/assets/133086327/1179449d-4b50-4feb-9fcc-87b17fbc201c">




#### To Test:

- yarn start:example2
- Click the Registration workflow and see the verify code screen page 



#### Any specific feedback you are looking for?

-
